.rubocop.yml

```
@@ -100,6 +101,168 @@ Packaging/RequireHardcodingLib:
 Packaging/RequireRelativeHardcodingLib:
   Enabled: true
 
+Performance:
+  Enabled: true
+
+Performance/AncestorsInclude:
+  Enabled: false
+
+Performance/ArraySemiInfiniteRangeSlice:
+  Enabled: false
+
+Performance/BigDecimalWithNumericArgument:
+  Enabled: true
+
+Performance/BindCall:
+  Enabled: true
+
+Performance/BlockGivenWithExplicitBlock:
+  Enabled: true
+
+Performance/Caller:
+  Enabled: true
+
+Performance/CaseWhenSplat:
+  Enabled: true
+
+Performance/Casecmp:
+  Enabled: false
+
+Performance/ChainArrayAllocation:
+  Enabled: false
+
+Performance/CollectionLiteralInLoop:
+  Enabled: true
+  Exclude:
+    - features/**/*
+    - spec/**/*
+
+Performance/CompareWithBlock:
+  Enabled: true
+
+Performance/ConcurrentMonotonicTime:
+  Enabled: true
+
+Performance/ConstantRegexp:
+  Enabled: true
+
+Performance/Count:
+  Enabled: true
+
+Performance/DeletePrefix:
+  Enabled: true
+
+Performance/DeleteSuffix:
+  Enabled: true
+
+Performance/Detect:
+  Enabled: true
+
+Performance/DoubleStartEndWith:
+  Enabled: true
+  IncludeActiveSupportAliases: true
+
+Performance/EndWith:
+  Enabled: true
+
+Performance/FixedSize:
+  Enabled: true
+
+Performance/FlatMap:
+  Enabled: true
+  EnabledForFlattenWithoutParams: false
+
+Performance/InefficientHashSearch:
+  Enabled: true
+
+Performance/IoReadlines:
+  Enabled: true
+
+Performance/MapCompact:
+  Enabled: false
+
+Performance/MapMethodChain:
+  Enabled: false
+
+Performance/MethodObjectAsBlock:
+  Enabled: true
+
+Performance/OpenStruct:
+  Enabled: true
+  Exclude:
+    - features/**/*
+    - spec/**/*
```

I didn't realize we use OpenStruct. We have two instances. I'd prefer that we remove them since we shouldn't be using them at all. I'll leave it up to you if you'd rather do that here or through a separate PR. It's just for two specs. 

For this PR though, at a minimum we should remove the `features` directory since there are no instances there and if there were any, we'd want to enforce this.

```suggestion
Performance/OpenStruct:
  Enabled: true
  Exclude:
    - spec/**/*
```

---

Will remove those here

---

Great thanks!